test/common/vmimages.py

```diff
@@ -114,9 +118,74 @@ def enough_disk_space():
     free = st.f_bavail * st.f_frsize / (1024*1024*1024)
     return free >= PRUNE_THRESHOLD_G;
 
+def get_refs():
+    """Return dictionary for available refs of the format {'rhel-7.4': 'ad50328990e44c22501bd5e454746d4b5e561b7c'}
+       Expects to be called from the top level of the git checkout
+    """
+    # get all remote heads and filter empty lines
+    # output of ls-remote has the format
+    #
+    # d864d3792db442e3de3d1811fa4bc371793a8f4f	refs/heads/master
+    # ad50328990e44c22501bd5e454746d4b5e561b7c	refs/heads/rhel-7.4
+    refs = filter(None, subprocess.check_output(["git", "ls-remote",  "--heads"]).split("\n"))
+    # filter out the "refs/heads/" prefix
+    # and generate a dictionary
+    prefix = "refs/heads"
+    return dict(map(lambda ref: [s[s.startswith(prefix) and len(prefix):] for s in reversed(ref.split())], refs))
+
+def get_image_links(ref):
+    """Return all image links for the given git ref
+       Expects to be called from the top level of the git checkout
+    """
+    # get all the links we have first
+    # trailing slash on GIT_IMAGE_PATH is important
+    git_path = GIT_IMAGE_PATH if GIT_IMAGE_PATH.endswith("/") else "{0}/".format(GIT_IMAGE_PATH)
```

Comment:
Well, you should keep the comment, of course. Or maybe just always append a `/` -- it's a constant after all, and two (or more) slashes work just fine.